templates/create_movies_form.html

```diff
@@ -6,4 +6,29 @@
 <h1 class="mb-5">Create Movie Rating</h1>
 <p class="mb-3">Create a movie rating below</p>
 <!-- TODO: Feature 2 -->
-{% endblock %}
+<div class="container">
```

Comment:
Overall the form looks good and includes all the required fields and I like the select option for stars instead of an input field